### ISSUE TITLE
when I am training, the validate mAP is very small ,is this normal?

### ISSUE THREAD

**Issue by tangxiaochu123230** [ultralytics/yolov5#91]
gpu_mem      GIoU       obj         cls       total        targets     img_size
3.15G          0.06724   0.05453  0.006653    0.1284         1       640

 Class      Images     Targets           P           R        mAP@.5     mAP@.5:.95
  all         306             551      0.0426       0.149      0.0571      0.0136

also the training loss nearly do not decrease ,keeping about 0.11

**Comment by tangxiaochu123230** [comment]
after training 50 epoches, my validate mAP do not improve any more,always keep about 0.02. I just do not know what is the problem

**Comment by glenn-jocher** [comment]
@tangxiaochu123230 you should git pull and train again. I just pushed several updates to help train custom datasets, including AutoAnchor, which automatically adapts your anchors to your data.

Also in the future, rather than showing one number or saying something is small (which means nothing), you should take a screenshot of your tensorboard or plot your results.txt file and upload the actual images here.

**Comment by AIFAN-Lab** [comment]
@glenn-jocher I just git pull the newest repo, including AutoAnchor,and train my own dataset
![image](https://user-images.githubusercontent.com/64510205/84994639-8d6d4b80-b17d-11ea-9afa-9442016d9873.png)

**Comment by parluo** [comment]
![image](https://user-images.githubusercontent.com/37440340/84994885-edfc8880-b17d-11ea-9aec-99b15564ae6c.png)
I think we suffer the same problem.
When I used the model to train the mask dataset, and as you see i got a low mAP,is there anything wrong? 
However，it did well in a self-made dataset which the object size is larger than the mask dataset two days before.

**Comment by parluo** [comment]
@glenn-jocher

**Comment by glenn-jocher** [comment]
@parluo @AIFAN-Lab @tangxiaochu123230 there was a bug https://github.com/ultralytics/yolov5/issues/117 in autoanchor assignment resolved yesterday, which is likely affecting you. Please `git pull` to receive the latest updates and try again.

If problems persist, you can skip autoanchor also using the command below. You're encouraged to trian with and without it, as it is still experimental. The guidance is that it will help lower BPR datasets (i.e. < 0.90) much more than high BPR datasets.

`python train.py --noautoanchor`

**Comment by jingenyan** [comment]
@tangxiaochu123230 i have the same problem and loading the pretrained model error ,it show the model parameters are different with each other
